content/docs/SUSHI/tips/_index.md

```diff
@@ -271,3 +271,34 @@ This IG provides [MyPatient] for patient information.
 
 {% include fsh-link-references.md %}
 ```
+
+## Choosing a SNOMED CT edition
+
+To specify a certain edition of SNOMED CT - for example to ensure terminology validation works correctly - do the following:
+
+1. Add a new `parameters` section to your `sushi-config.yaml`:
```

Comment:
It's possible that the config already has a parameters section, so perhaps this should be:
```md
1. Add or modify the `parameters` section in your `sushi-config.yaml` to specify a `path-expansion-params` parameter.
```